app.rb

```diff
@@ -0,0 +1,99 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+
+# rubocop:disable Style/GlobalVars
```

Comment:
- [ ] Your app works fine but I noticed that you disabled **Rubocop** a lot is not a good practice. Please remove all the **Rubocp disable** statements in your codebase and try to fix all the errors you are getting. Because ` Rubocop` helps developers to write clean Ruby code and adapt best practices when writing the code.  :+1: